content/ansible-getting-started/5-handlers/_index.md

```diff
@@ -9,14 +9,14 @@ Ansible can use conditionals to execute tasks or plays when certain conditions a
 
 To implement a conditional, the `when` statement must be used, followed by the condition to test. The condition is expressed using one of the available operators like e.g. for comparison:
 
-|      |                                                                        |
-| ---- | ---------------------------------------------------------------------- |
-| ==  | Compares two objects for equality.                                     |
-| !=  | Compares two objects for inequality.                                   |
-| >   | true if the left hand side is greater than the right hand side.        |
-| >=  | true if the left hand side is greater or equal to the right hand side. |
-| <   | true if the left hand side is lower than the right hand side.          |
-| <= | true if the left hand side is lower or equal to the right hand side.   |
+|     |     |
```

Comment:
Not super important but did you remove the "formatting" blanks on purpose?

Reply:
Yes, because they are ugly and not necessary for Markdown